.gitignore

```diff
@@ -1,3 +1,6 @@
+# client
+client/node_modules
+
```

Comment:
Could you add this `**/.DS_Store` for the mac users under the `# general` section and then run this command in the root of the project
`find . -name .DS_Store -print0 | xargs -0 git rm -f --ignore-unmatch` and then commit.

It will just remove the DS_Store files that have been committed already